types/smokehouse.d.ts

```diff
@@ -19,17 +19,28 @@
     diff?: Difference | null;
   }
 
-  export type ExpectedLHR = Pick<LH.Result, 'audits' | 'finalUrl' | 'requestedUrl' | 'runtimeError'>
+  interface ExpectedLHR {
+    audits: Record<string, any>;
```

Comment:
Is there a way to get all the audit ids? I tried something with `import('../lighthouse-core/config/default-config')` but couldn't get it to work.

Reply:
why is it divorced from `LH.Result` at all?

Reply:
Because we allow RegExps for any place a string is found, and `{length:1}` for arrays. Since the expectation files were changed to directly use this type now, it breaks if just `LH.Result` is used.